### PR TITLE
Feature/gemify

### PR DESCRIPTION
**Pull request by zorafuchs**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+### We love pull requests. Here's a quick guide.
+
+Fork, then clone the repo:
+```
+git clone git@github.com:your-username/renuo-cms-rails.git
```
Comment (lukaselmer) [inline CONTRIBUTING.md]: renuo-cms-rails? ;)

File: renuo-bin-check.gemspec [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'renuo/bin-check/version'
+
+Gem::Specification.new do |spec|
+  spec.name = 'renuo-bin-check'
+  spec.version = Renuo::BinCheck::VERSION
+  spec.authors = ['Zora Fuchs']
+  spec.email = ['zora.fuchs@renuo.ch']
+
+  spec.summary = 'The Renuo bin-check automates running programms to check code quality of a rails application'
```
Comment (lukaselmer) [inline renuo-bin-check.gemspec]: actually not only rails apps, but any app 😉 
i would also append the following: caching and parallel execution

File: renuo-bin-check.gemspec [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  spec.add_development_dependency 'rubocop',          '>= 0.39.0'
+  spec.add_development_dependency 'reek',             '>= 4.0.1'
+  spec.add_development_dependency 'rspec'
+  spec.add_development_dependency 'pry'
+  spec.add_development_dependency 'byebug'
+  spec.add_development_dependency 'pry-byebug'
+  spec.add_development_dependency 'simplecov'
+  spec.add_development_dependency 'awesome_print'
+  spec.add_development_dependency 'factory_girl'
+  # spec.add_development_dependency 'binding_of_caller'
```
Comment (lukaselmer) [inline renuo-bin-check.gemspec]: not needed :)

**Comment by lukaselmer** [comment]
we should do some small stuff, but otherwise it lgtm

File: .gitignore [hunk 4]
```diff
@@ -1,18 +1,103 @@
...
-/Gemfile.local.rb
-/coverage
-.byebug_history
-/.env
-/bin/endless-rspec
-/app/assets/javascripts/**/*.js.map
-/app/assets/javascripts/**/*.js
-.tsdrc
\ No newline at end of file
+### JetBrains template
```
Comment (lukaselmer) [inline .gitignore]: this may be a bit of an overkill. let's keep it simpler.

File: CONTRIBUTING.md (hunk 1 above)
Comment (zorafuchs) [inline CONTRIBUTING.md]: ubs 😳

File: .gitignore (hunk 4 above)
Comment (zorafuchs) [inline .gitignore]: so should I use the gitignore from before?

Comment (lukaselmer) [inline .gitignore]: yep, i would suggest so 👍

File: renuo-bin-check.gemspec [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  spec.homepage = 'https://github.com/renuo/renuo-bin-check'
+  spec.license = 'MIT'
+
+  spec.files = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.bindir = 'exe'
+  spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ['lib']
+  spec.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
+
+  spec.add_development_dependency 'rubocop',          '>= 0.39.0'
```
Comment (lukaselmer) [inline renuo-bin-check.gemspec]: Nice! :) you may also be a bit more specific here, e.g. like https://github.com/renuo/renuo-cms-rails/blob/master/renuo_cms_rails.gemspec#L28

Comment (zorafuchs) [inline renuo-bin-check.gemspec]: hmm which version should I use for which gem? just the newest one?

Comment (lukaselmer) [inline renuo-bin-check.gemspec]: yes, i would suggest that :)

File: renuo-bin-check.gemspec [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  spec.add_development_dependency 'reek',                         '>= 4.0.2'
+  spec.add_development_dependency 'rspec',                        '>= 3.4.0'
+  spec.add_development_dependency 'pry',                          '>= 0.10.3'
+  spec.add_development_dependency 'byebug',                       '>= 8.2.5'
+  spec.add_development_dependency 'pry-byebug',                   '>= 3.3.0'
+  spec.add_development_dependency 'simplecov',                    '>= 0.11.2'
+  spec.add_development_dependency 'awesome_print',                '>= 1.6.1'
+  spec.add_development_dependency 'factory_girl',                 '>= 4.7.0'
+  spec.add_development_dependency 'codeclimate-test-reporter',    '>= 0.5.0'
+end
```
Comment (lukaselmer) [inline renuo-bin-check.gemspec]: Nice!

**Comment by lukaselmer** [comment]
Nice, LGTM!

**Pull request merged**